go/storage/mkvs/checkpoint/checkpointer.go

```diff
@@ -285,17 +290,22 @@ func (c *checkpointer) worker(ctx context.Context) {
 		c.logger.Debug("storage checkpointer terminating")
 	}()
 
-	// Use a ticker to avoid checking for checkpoints too often.
-	ticker := time.NewTicker(c.cfg.CheckInterval)
-	defer ticker.Stop()
+	getCheckInterval := func() time.Duration {
```

Comment:
This is the same as:
https://github.com/oasisprotocol/oasis-core/blob/a9e863a7cc07c9662bfced2a473c0e3b67473a83/go/consensus/tendermint/full/helpers.go#L9

Let's move it somewhere under `go/common` and use it in both places.